Webapp/sources/blueprints/reaction_list/routes.py

```diff
@@ -5,12 +5,12 @@
 
 from sources.auxiliary import get_notification_number, get_workgroups
 
-from . import standard_landing_page_bp  # imports the blueprint of route
+from . import reaction_list_bp  # imports the blueprint of route
 from .reaction_list import get_reaction_list, get_scheme_list
 
 
 # Go to the delete profile page
-@standard_landing_page_bp.route("/delete_profile", methods=["GET", "POST"])
+@reaction_list_bp.route("/delete_profile", methods=["GET", "POST"])
 @login_required
 def delete_profile() -> Response:
```

Comment:
This is now duplicated as it's in the `delete_profile` blueprint - remove it from this one, as at least it makes sense in it's own blueprint for now rather than grouped with this one.

Reply:
Thanks for noticing. I will delete this one